tests/bearlib/languages/LanguageTest.py

```diff
@@ -27,3 +27,19 @@ def test_invalid_attribute(self):
     def test_attribute_list_empy(self):
         with self.assertRaisesRegexp(AttributeError, 'no available attribute'):
             self.lang_unknown.not_an_attribute
+
+
+class LanguagesTest(unittest.TestCase):
+
+    def setUp(self):
+        self.items = Languages([Language.Python == 3])
+        self.tuple = (Language['Python 3.3, 3.4, 3.5, 3.6'],)
+
+    def tearDown(self):
+        pass
+
+    def test_language_tuple(self):
```

Comment:
You don't need to include the term 'language_' in every method name, since `Languages` is already in the class name. Or in other words: Please remove ;)